xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/hierarchy_macros.vm

```diff
@@ -146,7 +149,17 @@
       #set ($breadCrumbParameters = "$breadCrumbParameters data-${option.toLowerCase()}='$escapetool.xml($options.get($option))'")
     #end
   #end
-  <ol #if("$!options.id" != '')id="$options.id"#end class="$breadCrumbClassNames" $breadCrumbParameters><li
+  ## Set as a navigation role because it contains mostly links
+  ## Except if the 'navigation' option is set to false, for special cases where a navigation would clutter
+  ## See discussion on https://forum.xwiki.org/t/ui-accesibility-breadcrumbs-in-the-document-index/12866
+  <#if($!options.navigation == false)div#{else}nav#end aria-label=
```

Comment:
Citing https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label:

> Note: While aria-label is allowed on any element that can have an accessible name, in practice, aria-label is only supported on interactive elements, widgets, landmarks, images and iframes.

It therefore doesn't make any sense to set an `aria-label` when the element is a `div`. Further, it is therefore also not useful to introduce a ton of new translation keys to set labels on elements that cannot be labelled.

I'm wondering if it wouldn't be better to leave the `#hierarchy` macro alone to avoid that some code introduces accidentally a ton of landmarks on a page and to instead wrap the hierarchy at the top of the page in a `<nav>` element. Alternatively, we could say that the parameter for the label is named `navigationLabel` and that the presence of this parameter triggers the `<nav>` element. But still, the wrapping `<div>` is completely useless and any caller that wants the navigation role could also simply wrap the breadcrumbs in an appropriately labelled `<nav>` element.